circleci/x86_64/Dockerfile

```diff
@@ -4,5 +4,5 @@ FROM $IMAGE_NAME
 LABEL authors="Hajime Tazaki <thehajime@gmail.com>, Octavian Purdila <tavi@cs.pub.ro>"
 
 RUN sudo apt-get update && \
-    sudo apt-get install -y libfuse-dev libarchive-dev linux-headers-4.4.0-97-generic valgrind libnuma-dev && \
+    sudo apt-get install -y libfuse-dev libarchive-dev linux-headers-generic valgrind libnuma-dev && \
```

Comment:
lklfuse recently moved to libfuse3 - I'm not sure about Ubuntu / Debian, but I think we should add  `libfuse3-dev` here. ` libfuse-dev` could then be dropped